docs/source/setup/manual_way.rst

```diff
@@ -1,55 +1,60 @@
 Manual Way
 ************
+### Quick install
+Its simplicity makes SQLite extremely useful for development. It speeds up the setup of the development environment.
```

Comment:
This part (SQL) is not relevant for installation process.

Reply:
> is not relevant for installation process.
@ciur , let me explain.  You offer 3 different installation options. I found out via Email exchange why you use SQLite here instead of Postgre. I think it is nothing more but good practice to explain that in advance.